python/ray/tests/test_placement_group.py

```diff
@@ -23,6 +23,59 @@ def method(self, x):
         return x + 2
 
 
+def test_placement_group_implicit_resource(ray_start_cluster):
+    @ray.remote(num_cpus=0)  # No resource specified for this actor.
```

Comment:
We can just remove num_cpus=0 (since that's the default)

Reply:
updated.